### PR TITLE
grant creation preview step

### LINKED ISSUE #10154: Create grant preview step
**User Story:**
As a grant owner
I want to preview what my grant details will look like
So that I can be sure that my information is correct before I submit my grant for approval

**Acceptance Criteria**
GIVEN that I am on gitcoin.co/grants/new
WHEN I look at the top of the page
THEN I see a four-step progress bar

GIVEN that I am on step 3 of gitcoin.co/grants/new
WHEN I look at the body of the page
THEN I see enabled Back and Next buttons instead of the Edit / Confirm buttons

GIVEN that I am on step 3 of gitcoin.co/grants/new AND have entered in the required fields
WHEN I click on Next
THEN I am brought to step 4 of the grant creation flow

GIVEN that I am on step 3 of gitcoin.co/grants/new
WHEN I click on Back
THEN I am brought to step 2 of the grant creation flow

GIVEN that I am on step 4 of gitcoin.co/grants/new
WHEN I look at the body of the page
THEN I see a preview of my grant's detail page
AND I see the Edit / Confirm buttons

GIVEN that I am on step 4 of gitcoin.co/grants/new
WHEN I click the Confirm button
THEN my grant is submitted for approval
AND I am redirected to my grant's detail page

GIVEN that I am on step 4 of gitcoin.co/grants/new
WHEN I click the Edit button
THEN I am brought back to step 3

**Product & Design Links:**
Figma — [Ticketed: Create a Grant](https://www.figma.com/file/8QjqOzDBd4HiE37YqcocK7/?node-id=163%3A3717) Section 2

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**

### PR DESCRIPTION
**Pull request by chibie**
##### Description

This PR implements a preview step for the grant creation multi-step form.

##### Refers/Fixes

closes #10154 

##### Testing

Screenshot 2022-04-07 at 00 41 18

### CODE REVIEW

File: app/grants/templates/grants/grant-preview.html [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+            <template v-for="item in grant_tags">
+              <a v-if="item.is_eligibility_tag"
+                class="badge badge-pill bg-tags text-primary py-1 px-2 font-smaller-2 font-weight-normal mr-2"
+              >
+                [[item.name]]
+              </a>
+            </template>
+
+            <!-- Grant Discovery Tags -->
+            <template v-for="item in grant_tags">
```
Comment (thelostone-mc) [inline app/grants/templates/grants/grant-preview.html:44]: we could try grouping L44 and L35 into the same section

File: app/grants/templates/grants/grant-preview.html [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+        </div>
+
+      </div>
+
+      <div class="row border-bottom font-smaller-1 py-4 mx-0 mt-1 mb-4 ">
+
+        <!-- Grant Link -->
+        <div class="col-12 col-md-5 col-md-auto mb-3">
+          <i class="fas fa-link text-grey-400"></i>
+          <a target="_blank" :href="form.reference_url"
```
Comment (thelostone-mc) [inline app/grants/templates/grants/grant-preview.html:63]: let's add in noopener noreferrer

File: app/grants/templates/grants/grant-preview.html [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+          <div v-for="member in teamMembers" :key="member.id" class="d-flex flex-column align-items-center px-2 py-3" v-if="member?.text">
+            <b-img-lazy class="rounded-circle bg-light" width="50" height="50" :src="member.avatar_url" :alt="member.text"></b-img-lazy>
+            <a :href="`/profile/${member.text}`" class="mt-2">[[member.text]]</a>
+          </div>
+        </div>
+      </div>
+
+    </div>
+
+  </div>
```
[the remark this answers is not in the document]
Comment (schultztimothy) [reply app/grants/templates/grants/grant-preview.html:360]: This would be great! 🚀

[the remark this answers is not in the document]
Comment (chibie) [reply app/grants/templates/grants/grant-preview.html:360]: @thelostone-mc @schultztimothy it's not really duplicated as the js action handling was stripped + some of the state data is different

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**